src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { memo } from "react";
+
+const Footer = memo(() => {
```

Comment:
Why you use here memo ? and where types for this component ? and better if you use short form without return